src/web/web-service/frontend/res/js/app.js

```diff
@@ -372,5 +373,487 @@
                 ev.target.disabled = false;
             });
         };
+
+        $scope.restServerPort = '8081';
```

Comment:
@tttttangTH I don't think this is what we finally agreed on, right?

Reply:
Just exists for the time being because now we still use OTBR-Web to serve static files, it is removed in next PR.

Reply:
okay, makes sense.